imgui-gfx-renderer/src/lib.rs

```diff
@@ -215,6 +215,8 @@ impl<R: Resources> Renderer<R> {
         self.bundle.slice.start = 0;
         for cmd in draw_list.cmd_buffer {
             // TODO: check cmd.texture_id
+            // TODOK: here we can change the texture based on
+            // self.bundle.data.texture = imgui.retrieve_texture(cmd.texture_id)?;
```

Comment:
Last time I try, you could do this and it worked:

```rust
self.bundle.data.tex.0 = imgui.retrieve_texture(cmd.texture_id)?
                .clone();
```

But it would really slow down the program, so I would not recommend it.